### PR TITLE
feat(mergify): Update dashboard with new metrics, add service check for GitHub ratelimit

### PR DESCRIPTION
**Pull request by Greesb**
### What does this PR do?

- Updated the default dashboard with the metrics, added and modified, in 0.1.0 and 1.0.0
- Added a `WARNING` service check that will be triggered when the user or organization, that the tokens belongs to, is rate limited on GitHub

### Motivation

I really like pie charts

### Review checklist

- [X] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [X] Feature or bugfix has tests
- [X] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

N/A

### CODE REVIEW

**Comment by Greesb** [comment]
@sileht

**Review by sileht — APPROVED** [review]

File: mergify/assets/dashboards/mergify_overview.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "Mergify overview",
+  "title": "Mergify integration overview",
```
Comment (sileht) [inline mergify/assets/dashboards/mergify_overview.json:2]: Not sure about this change

Comment (Greesb) [reply mergify/assets/dashboards/mergify_overview.json]: Removed it

**Review by FlorentClarret — APPROVED** [review]
Sorry for the delay.

Looks good to me. Your modifications (new released version and the dashboards) will be available soon after the PR gets merged.

**Comment by Greesb** [comment]
> Sorry for the delay.
> 
> Looks good to me. Your modifications (new released version and the dashboards) will be available soon after the PR gets merged.

Thanks @FlorentClarret !

**Pull request merged**